pkg/engine/project.go

```diff
@@ -21,7 +21,6 @@ import (
 	"strings"
 
 	"github.com/pkg/errors"
-
```

Comment:
This separation into three blocks is intentional in our style across the codebase.

Reply:
Fixed!